### DISCUSSION TITLE
What method can be used to stop its fall?

### DISCUSSION THREAD

**Discussion by MingSongJiang** [InfinyTech3D/SofaUnity#119] [Help - Using SofaUnity] [answered]
Hello,

What method can be used to stop its fall? Because mass is required, when the cut part is not blocked, the cut part falls to the ground, and the ground can prevent the part that falls to the ground from continuing to fall.

Thanks!

**Comment by epernod** [comment, marked as answer]
Hi,

you can either put a floor defined by a mesh with collision element or you could also use a planeForceField. There is an example in the LiverInteraction scenes
